applications/mne_analyze/libs/anShared/Model/annotationmodel.cpp

```diff
@@ -864,17 +864,29 @@ void AnnotationModel::setGroupName(int iGroupIndex,
 
 //=============================================================================================================
 
-const QString& AnnotationModel::getGroupName(int iGroupIndex)
+QString AnnotationModel::getGroupName(int iGroupIndex)
```

Comment:
Do we still need this function or is it obsolete due to the new `getGroupNameFromList`?

Reply:
Currently we are not using it, but they get data in a different way. The old function, getGroupName, gets the name based on the group's map key, while the new one just gets a list of all groups and selects from this list, disregarding the map key. Should we remove it for now and if we need that functionality later we just re-implement it?

Reply:
ok. so in the second (new) function the function Parameter should maybe be renamed to `iGroupListIndex`? And in the old function maybe we should use `iGroupMapKey`?